src/Tests/Microsoft.NET.Sdk.Razor.Tests/StaticWebAssetsIntegrationTest.cs

```diff
@@ -193,6 +199,121 @@ public void BuildProjectWithReferences_GeneratesJsonManifestAndCopiesItToOutputF
                 intermediateOutputPath);
         }
 
+        [Fact]
+        public void BuildProjectWithReferences_WorksWithStaticWebAssetsV1ClassLibraries()
```

Comment:
Do we need an additional set of scenarios for a 3.1 app targeting ns2.1 library? We have some MVC tests, but I don't think we have any that cover SWA

Reply:
Those all go through the old code paths and don't mix old code with new code, so I think we can file an issue and cover them later.

Reply:
Do we file it here or on the asp.net core repo?

Reply:
dotnet/aspnetcore. We don't triage issues here, so they're bound to get lost .